### ISSUE TITLE
MCP23017 missing from lbr

### ISSUE THREAD

**Issue by WookieWrath** [adafruit/Adafruit-Eagle-Library#21]
I am trying to find the MCP23017 in the Eagle Library but I am unable to find it.
There are comments stating it was added but its not there.

**Comment by xurtej** [comment]
couldnt find it either can anyone help with that ?

**Comment by JensGrabner** [comment]
Yes,  here you found:  https://github.com/adafruit/Adafruit-Eagle-Library
